### PR TITLE
Ex. 2: FootBook

### PR DESCRIPTION
**Pull request by KinkyHairCoder**
- Created separate components for header, posts, groups, and footer.

### CODE REVIEW

File: src/Student1/Ex2/Nav.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+* {
```
Comment (mburolla) [inline src/Student1/Ex2/Nav.scss]: This "select all" selector is duped in `Post.scsss`, so we should remove one.

File: src/Student1/Ex2/Header.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+* {
```
Comment (mburolla) [inline src/Student1/Ex2/Header.scss]: It's also here.

File: src/Student1/Ex2/Groups.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+* {
```
Comment (mburolla) [inline src/Student1/Ex2/Groups.scss]: And here.

File: src/Student1/Ex2/Footer.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+* {
```
Comment (mburolla) [inline src/Student1/Ex2/Footer.scss]: And here.

File: src/Student1/Ex2/Nav.scss (hunk 1 above)
Comment (KinkyHairCoder) [reply src/Student1/Ex2/Nav.scss]: Made the Changes

**Comment by mburolla** [comment]
Good job, now you are ready for the hard stuff.  :-)

**Pull request merged**